Resources/config/odm.xml

```diff
@@ -16,6 +16,10 @@
         <parameter key="doctrine_phpcr.odm.cache.memcache_host">localhost</parameter>
         <parameter key="doctrine_phpcr.odm.cache.memcache_port">11211</parameter>
         <parameter key="doctrine_phpcr.odm.cache.memcache_instance.class">Memcache</parameter>
+        <parameter key="doctrine_phpcr.odm.cache.memcached.class">Doctrine\Common\Cache\MemcachedCache</parameter>
+        <parameter key="doctrine_phpcr.odm.cache.memcached_host">localhost</parameter>
+        <parameter key="doctrine_phpcr.odm.cache.memcached_port">11211</parameter>
```

Comment:
does this rely on global parameter overwriting or is there some generic magic in the Configuration for it?